swarm/api/config.go

```diff
@@ -58,7 +58,7 @@ type Config struct {
 	Port                 string
 	PublicKey            string
 	BzzKey               string
-	NodeID               string
+	Enode                *enode.Node `toml:"-"`
```

Comment:
Just for curiosity, what do we use `toml` configuration language for? I'm just surprised by it's presence and also by that other field don't require the annotation.

Reply:
We use it for stored node configs.